docs/framework/network-programming/network-programming-samples.md

```diff
@@ -16,60 +16,49 @@ ms.workload:
   - "dotnet"
 ---
 # Network Programming Samples
-This section contains descriptions and links to downloadable network programming samples that use classes in the <xref:System.Net>, <xref:System.Net.Cache>, <xref:System.Net.Configuration>, <xref:System.Net.Mail>, <xref:System.Net.Mime>, <xref:System.Net.NetworkInformation>, <xref:System.Net.PeerToPeer>, <xref:System.Net.Security>, <xref:System.Net.Sockets>, and related namespaces.  
+This section contains descriptions and links to downloadable network programming samples that use classes in the <xref:System.Net>, <xref:System.Net.Cache>, <xref:System.Net.Configuration>, <xref:System.Net.Mail>, <xref:System.Net.Mime>, <xref:System.Net.NetworkInformation>, <xref:System.Net.PeerToPeer>, <xref:System.Net.Security>, <xref:System.Net.Sockets>, and related namespaces.
+Note: all samples are available in the .NET Framework SDK version 2.0 Samples Download, which was released for the .NET Framework 2.0 and may be dated.
```

Comment:
This runs into a single paragraph, and so it's easily overlooked. Add a blank line between lines 19 and 20.

But this is best done as an alert box so that it can be more noticeable, and we'd want to include the link to the samples. So:

\> [!NOTE]
\> All samples are available in the \[.NET Framework SDK version 2.0 Samples Download\]\(https://www.microsoft.com/en-us/download/confirmation.aspx?id=22181), which was released for the .NET Framework 2.0 and may be dated.